scripts/pi-hole/js/formatter.mjs

```diff
@@ -0,0 +1,18 @@
+/* Pi-hole: A black hole for Internet advertisements
+ *  (c) 2024 Pi-hole, LLC (https://pi-hole.net)
+ *  Network-wide ad blocking via your own hardware.
+ *
+ *  This file is copyright under the latest version of the EUPL.
+ *  Please see LICENSE file for your rights under this license. */
+
+export function toPercent(number, fractionDigits = 0) {
```

Comment:
To stay consistent with the current code base, this should probably be in `utils.js` rather than it's own file. Is there a reason you chose to use a separate file?

Reply:
It is also used in index.js. Can I call it from utils.js?

Reply:
Yes, `utils.js` is sourced in `header.lp`, so can be used globally.

Also worth noting that we're not currently using any fancy javascript frameworks in this project